pkg/crc/cluster/cluster.go

```diff
@@ -200,6 +200,8 @@ func addPullSecretToInstanceDisk(sshRunner *ssh.SSHRunner, pullSec string) error
 	if err != nil {
 		return err
 	}
+	_, _ = sshRunner.RunPrivate("sudo chmod 600 /var/lib/kubelet/config.json")
```

Comment:
@cfergeau I would prefer to have a single command for ssh, something like following so we also don't ignore the error.
```patch
diff --git a/pkg/crc/cluster/cluster.go b/pkg/crc/cluster/cluster.go
index e17235d..e0419cd 100644
--- a/pkg/crc/cluster/cluster.go
+++ b/pkg/crc/cluster/cluster.go
@@ -196,7 +196,7 @@ func AddProxyConfigToMarketplaceOperator(oc oc.OcConfig, proxy *network.ProxyCon
 }
 
 func addPullSecretToInstanceDisk(sshRunner *ssh.SSHRunner, pullSec string) error {
-       _, err := sshRunner.RunPrivate(fmt.Sprintf("cat <<EOF | sudo tee /var/lib/kubelet/config.json\n%s\nEOF", pullSec))
+       _, err := sshRunner.RunPrivate(fmt.Sprintf("sudo install -m 600 /dev/null /var/lib/kubelet/config.json && cat <<EOF | sudo tee /var/lib/kubelet/config.json\n%s\nEOF", pullSec))
        if err != nil {
                return err
        }
```

Reply:
Ignoring the error was intentional as it's no big deal if we fail to change the permissions, it's only causing a warnings in some operator's logs. I'll try your suggestion though as I'm fine either way.